frontend/src/components/cluster/Overview.tsx

```diff
@@ -126,7 +130,11 @@ function EventsSection() {
               ]
             : []
         }
-        data={events}
+        data={
+          eventsFilter
+            ? events.filter((event: Event) => event.metadata.uid === eventsFilter)
+            : events
+        }
```

Comment:
I'm not sure this is the best way as it will only show one event without chances to seeing the rest without going to the view without the filter. Could we maybe add the event ID as a special search field and then use that views search entry?